src/MockGenerateHelper.ts

```diff
@@ -257,6 +258,34 @@ export class MockGenerateHelper {
         return { propertyName, value: `" // TODO: Wrong dictionary value",` };
     }
 
+    getAdditionalPropertiesMock({ propertyName, additionalProperties }: GetAdditionalPropertiesProps): MockArrayProps {
+        let result: string | Function | undefined = undefined;
+
+        switch (additionalProperties?.type) {
+            case DataTypes.Integer:
+                result = () => casual.integer(0, 100);
+                break;
+            case DataTypes.Number:
+                result = () => casual.double(0, 100);
+                break;
+            case DataTypes.String:
+                result = () => casual.string;
+                break;
+            case DataTypes.Boolean:
+                result = 'true';
+                break;
+            case DataTypes.Array:
+                result = '[]';
+                break;
+            default: {
+                result = ' // Error: Invalid additionalProperties.type';
+                break;
+            }
+        }
+
+        return { propertyName, value: result };
+    }
```

Comment:
will not do as I did for the types, so no improvement / cleanup / refact. just this new method to write mock when additional properties is provided without a dictionary.